### DISCUSSION TITLE
[RFC] Recipe Auth Tail for Authentication Design with Tailwind CSS 2.x

### DISCUSSION THREAD

**Discussion by Anjianto** [blitz-js/blitz#2310] [RFCs]
# Problem

Authentication without design bad for apps and not all can design it. If someone can design it fell pain to design and implement to code.

# Solution

Auth Tail make it easy. Authentication with Tailwind CSS and include with beautiful responsive authentication. You will get auth page with less CSS and make it faster as development as production. Auth Tail give you 3 form choices with Blitz give for you

# The page:
## Sign In
![Screenshot from 2021-05-07 05-05-44](https://user-images.githubusercontent.com/61521141/117371439-15715680-aef2-11eb-891f-8e73a68404bc.png)

## Sign Up
![Screenshot from 2021-05-07 05-05-49](https://user-images.githubusercontent.com/61521141/117371452-1acea100-aef2-11eb-9e5f-d5ed5fbfad6d.png)

## Forgot Password
![Screenshot from 2021-05-07 05-05-59](https://user-images.githubusercontent.com/61521141/117371463-1e622800-aef2-11eb-8ac5-ee87cc7e96ed.png)

## Reset Password
![Screenshot from 2021-05-07 05-06-16](https://user-images.githubusercontent.com/61521141/117371478-2326dc00-aef2-11eb-86d5-ab46da1234fb.png)

**Comment by flybayer** [comment]
You want to make a recipe for this?
